### DISCUSSION TITLE
can't run `ord preview` on signet

### DISCUSSION THREAD

**Discussion by felipelincoln** [ordinals/ord#1565] [Q&A]
I have signet `bitcoind` running, I am able to use commands like `ord -s wallet balance`, `ord -s wallet send`, `ord -s inscribe`, ...

but when I try starting the preview server I get:
```
ord -s preview
```
```
...
Listening on http://0.0.0.0:80 ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ 0/103
error: Permission denied (os error 13)
```

If I try using `sudo` the server starts but on `regnet`.

```
sudo /home/ubuntu/bin/ord -s preview
```

How can I make it work on `signet`?

**Comment by casey** [comment]
Thanks for the issue!

`ord preview` does not run on `signet`, it always spins up a local regtest node and inscribes to it. Signet is a public test net, so ideally a local regtest node should be used for previewing.
